mw_jwt.go

```diff
@@ -335,6 +341,12 @@ func (k *JWTMiddleware) processCentralisedJWT(r *http.Request, token *jwt.Token)
 				log.WithError(err).Error("Could not apply new policy from JWT to session")
 				return errors.New("Key not authorized: could not apply new policy"), http.StatusForbidden
 			}
+			//override session expiry with JWT if it is longer lived
+			if k.Spec.JWTExpiryOverridesPolicy {
+				if int64(claims["exp"].(float64)) > session.Expires {
```

Comment:
Why this check is needed, and why not to always set expiration form JWT token if it is not zero?